apps/users/templates/users/login_form.html

```diff
@@ -1,4 +1,6 @@
-<form method="post" action="{{ action }}" class="{% if is_ajax %}ajax-submit {% endif %}listing-footer">
+{% set login_source_form = switch_is_active('fxa-auth') and not form.errors %}
```

Comment:
it's weird to me that this is also conditional on an error free form. Could you do the error logic a different way?